### ISSUE TITLE
Some bullet colors are hard to see

### ISSUE THREAD

**Issue by LeastCreative** [CodingTrain/Asteroids#24]
maybe a randomly generated color with at least one brightish value would be cool

**Comment by MunWolf** [comment]
We could hard code it so that when we generate a color we take the highest value, check if its at or above a certain threshold and if not we make it equal to the threshold.

So if the threshold was 192 then (150, 60, 120) would become (192, 60, 120) but (100, 212, 120)
would stay the same.

**Comment by meiamsome** [comment]
Probably a good idea to switch colour mode to HSV or something so we can linit the range easily.

**Comment by MunWolf** [comment]
Yeah was reading up on that (HSV), thought I remembered a way to make colors of a certain brightness.

**Comment by meiamsome** [comment]
Offering up this for anyone who wants to change the colour mode to HSV and alter the colour system.
